src/draive/conversation/completion/types.py

```
@@ -0,0 +1,53 @@
+from collections.abc import AsyncIterator
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)